src/library_html5.js

```diff
@@ -2705,6 +2705,7 @@ var LibraryHTML5 = {
     return requestAnimationFrame(tick);
   },
 
+  emscripten_date_now__sig: 'd',
   emscripten_date_now: function() {
     return Date.now();
   },
```

Comment:
I'd rather keep the version in `src/library.js` .. Date.now() exists under node too so its not html-specific.

Reply:
But I guess this is declared in `emscripten/html5.h`... so maybe can leave it for now and possibly move in future.